### PR TITLE
Add Installation Info

### PR DESCRIPTION
**Pull request by b3u**
Added OS specific instructions for installation.
This was sparked by my confusion after downloading Synfig on Ubuntu for the first time.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,6 +26,17 @@ Old autotools instructions can be found [here][autotools].
 We're currently in the [process of switching to CMake build system][cmake] and
 you're welcome to take part in testing and improving it.
 
+#### Debian/Ubuntu
+1. Download the appimage file from the website.
+2. In your terminal, run `sudo apt-get install synfigstudio`
```
[the remark this answers is not in the document]
Comment (b3u) [reply README.md]: That's how I did it on my machine. Was it incorrect? (It worked)

File: README.md [hunk 2]
```diff
@@ -26,6 +26,17 @@ Old autotools instructions can be found [here][autotools].
...
+
+#### Windows
+1. Download the `.exe` file from the website.
+2. Run the file. A wizard will then guide you through the rest of the installation.
+
+#### Mac OS
+1. Download the `.dmg` file from the website.
+2. Run the file. A wizard will then guide you through the rest of the installation.
```
[the remark this answers is not in the document]
Comment (b3u) [reply README.md]: My mistake. It was an assumption. I don't know macOS

**Comment by b3u** [comment]
I've committed the changes you've requested.

**Comment by ice0** [comment]
Merged. Thank you!

**Comment by b3u** [comment]
Thanks, man!

**Pull request merged**